test/matrix.lean

```diff
@@ -70,18 +70,15 @@ begin
   ring
 end
 
+local attribute [instance, priority 101] add_group.to_has_involutive_neg
+
 example {α : Type*} [comm_ring α] (A : matrix (fin 3) (fin 3) α) {a b c d e f g h i : α} :
         matrix.det ![![a, b, c], ![d, e, f], ![g, h, i]] =
           a * e * i - a * f * h - b * d * i + b * f * g + c * d * h - c * e * g :=
 begin
-  simp [matrix.det_succ_row_zero, fin.sum_univ_succ],
-  /-
-  Try this: simp only [det_succ_row_zero, fin.sum_univ_succ, neg_mul, cons_append,
-  mul_one, fin.default_eq_zero, fin.coe_zero, cons_vec_bit0_eq_alt0, one_mul, cons_val_one,
-  cons_vec_alt0, fin.succ_succ_above_one, fin.coe_succ, univ_unique, minor_apply, pow_one,
-  fin.zero_succ_above, fin.succ_zero_eq_one, fin.succ_succ_above_zero, nat.neg_one_sq,
-  finset.sum_singleton, cons_val_zero, cons_val_succ, det_fin_zero, head_cons, pow_zero]
-   -/
+  -- Previously we could do `simp [matrix.det_succ_row_zero, fin.sum_univ_succ]`, but that now times
+  -- out.
+  rw det_fin_three,
```

Comment:
I think that this example was here to show how to compute the determinant of any matrix, not only 3*3. Can we fix the timeout in some other way?

Reply:
I don't really understand why the timeout is happening I'm afraid. Do you think we need to get to the bottom of it before merging, or are you ok with just going ahead with the change anyway.